src/Parameters/CreateMeetingParameters.php

```diff
@@ -31,780 +97,309 @@ class CreateMeetingParameters extends MetaParameters
     /**
      * @var string
      */
-    private $meetingId;
+    protected $name;
 
     /**
      * @var string
      */
-    private $meetingName;
+    protected $meetingID;
 
     /**
      * @var string
      */
-    private $attendeePassword;
+    protected $attendeePW;
 
     /**
      * @var string
      */
-    private $moderatorPassword;
+    protected $moderatorPW;
 
     /**
      * @var string
      */
-    private $dialNumber;
-
-    /**
-     * @var int
-     */
-    private $voiceBridge;
+    protected $welcome;
 
     /**
      * @var string
      */
-    private $webVoice;
+    protected $dialNumber;
 
     /**
      * @var string
      */
-    private $logoutUrl;
+    protected $voiceBridge;
 
     /**
      * @var int
      */
-    private $maxParticipants;
+    protected $maxParticipants;
 
     /**
-     * @var bool
-     */
-    private $record;
-
-    /**
-     * @var bool
+     * @var string
      */
-    private $autoStartRecording;
+    protected $logoutUrl;
 
     /**
      * @var bool
      */
-    private $allowStartStopRecording;
+    protected $record;
 
     /**
      * @var int
      */
-    private $duration;
+    protected $duration;
 
     /**
-     * @var string
+     * @var boolean
      */
-    private $welcomeMessage;
+    protected $isBreakout;
 
     /**
      * @var string
      */
-    private $moderatorOnlyMessage;
+    protected $parentMeetingID;
 
     /**
-     * @var bool
+     * @var int
      */
-    private $webcamsOnlyForModerator;
+    protected $sequence;
 
     /**
-     * @var string
+     * @var boolean
      */
-    private $logo;
+    protected $freeJoin;
 
     /**
      * @var string
      */
-    private $copyright;
+    protected $moderatorOnlyMessage;
 
     /**
      * @var bool
      */
-    private $muteOnStart;
+    protected $autoStartRecording;
 
     /**
      * @var bool
      */
-    private $lockSettingsDisableCam;
+    protected $allowStartStopRecording;
 
     /**
      * @var bool
      */
-    private $lockSettingsDisableMic;
+    protected $webcamsOnlyForModerator;
 
     /**
-     * @var bool
+     * @var string
      */
-    private $lockSettingsDisablePrivateChat;
+    protected $logo;
 
     /**
-     * @var bool
+     * @var string
      */
-    private $lockSettingsDisablePublicChat;
+    protected $bannerText;
 
     /**
-     * @var bool
+     * @var string
      */
-    private $lockSettingsDisableNote;
+    protected $bannerColor;
 
     /**
-     * @var bool
+     * @var string
      */
-    private $lockSettingsHideUserList;
```

Comment:
https://github.com/bigbluebutton/bigbluebutton.github.io/issues/211 looks like the api specs on the website are not up to date. We should not remove this setting

Reply:
After looking at [ApiParams.java](https://github.com/bigbluebutton/bigbluebutton/blob/develop/bbb-common-web/src/main/java/org/bigbluebutton/api/ApiParams.java) it seams like there are more "hidden" options. I think I removed also `webVoice`. Maybe we should first try to fix the documentation, because maybe they deprecated some options.